content/docs/use-cases/shared-development-server.md

```diff
@@ -55,13 +55,21 @@ your <abbr>project</abbr>:
 $ dvc cache dir /home/shared/dvc-cache
 ```
 
-And tell DVC to set group permissions on newly created or downloaded cache
-files:
+External cache configuration:
 
 ```dvc
 $ dvc config cache.shared group
+$ dvc config cache.type "reflink,symlink,hardlink,copy"
```

Comment:
would be good to include here links - to the links guide and to the relevant section in the dvc config

also, not clear if `reflink` is a very meaningful option here, tbh ... it would work only in a very specific setup - may be we can mention that, but I would not expect that it would become commonly used tbh

Reply:
Linked to file links guide and moved the link to `config cache` closer to this p. Removed `reflink`

Reply:
What about `copy`? It's a bit unclear why we leave it in if we want to prevent copying.

Reply:
yep, sounds good to me